src/test/java/de/tum/in/www1/artemis/ModelingExerciseIntegrationTest.java

```diff
@@ -103,7 +114,10 @@ public void testGetModelingExerciseStatistics_asTA() throws Exception {
     @WithMockUser(username = "instructor1", roles = "INSTRUCTOR")
     public void testCreateModelingExercise_asInstructor() throws Exception {
         ModelingExercise modelingExercise = modelingExerciseUtilService.createModelingExercise(classExercise.getCourse().getId());
-        request.post("/api/modeling-exercises", modelingExercise, HttpStatus.CREATED);
+        ModelingExercise receivedModelingExercise = request.postWithResponseBody("/api/modeling-exercises", modelingExercise, ModelingExercise.class, HttpStatus.CREATED);
```

Comment:
also add a test, where you invoke PUT after POST and where you have changed some of the criteria (i.e. add a new one, change an existing one, delete an existing one) and the instructions